src/components/MessageList/MessageList.js

```diff
@@ -254,21 +258,23 @@ const MessageList = (props) => {
           </ErrorNotification>
         )}
       </View>
-      {// Mask for edit state
-      editing && disableWhileEditing && (
-        <TouchableOpacity
-          collapsable={false}
-          onPress={clearEditingState}
-          style={{
-            backgroundColor: 'black',
-            height: '100%',
-            opacity: 0.4,
-            position: 'absolute',
-            width: '100%',
-            zIndex: 100,
-          }}
-        />
-      )}
+      {
+        // Mask for edit state
+        editing && disableWhileEditing && (
+          <TouchableOpacity
+            collapsable={false}
+            onPress={clearEditingState}
+            style={{
+              backgroundColor: 'black',
+              height: '100%',
+              opacity: 0.4,
+              position: 'absolute',
```

Comment:
obviously this was already in there before but position absolute doesn't seem like it'd do anything here with the height and width and zIndex set